src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+                            </MenuHeaderBlock>
+                        </ContainerHeder>
+                    }
+                    {width <= 480 && <>                 {/* --------------- Mobile ------------- */}
+                        <ContainerHeder>
+                            <LeftBlock>
+                                <LogoHead />
+                                {location.pathname === '/home' &&
+                                    <PointsBalance>
+                                        Баланс баллов:
+                                        <NumberBalance>0</NumberBalance>
+                                    </PointsBalance>
+                                }
+                            </LeftBlock>
+                            <MobileHamburgerContainer onClick={toggle} >
+                                <MobileHamburgerLine />
+                            </MobileHamburgerContainer>
+                        </ContainerHeder>
+                        <MobileHamburgerbackground style={{ transform: isOpen ? "translateX(0%)" : "translateX(120%)", opacity: isOpen ? "1" : "0.3" }}>
+                            <MobileHamburgerCrossBox onClick={toggle}>
+                                <MobileHamburgerCross>
+                                    <MobileHamburgerCross1 />
+                                </MobileHamburgerCross>
+                            </MobileHamburgerCrossBox>
+                            <NavLink to="login" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenuUp>Авторизация</MobileHamburgerSectionMenuUp>
+                            </NavLink>
+                            <NavLink to="contacts" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenu>Контакты</MobileHamburgerSectionMenu>
+                            </NavLink>
+                        </MobileHamburgerbackground>
+                    </>
+                    }
+                </>
+            }
+            {UserName && // ------------------------------------------------ USER ---------------
+                <>
+                    {width >= 870 &&                     // --------------- Desktop -------------
+                        <ContainerHeder>
+                            <LeftBlock>
+                                <LogoHead />
+                                {location.pathname === '/home' &&
+                                    <PointBalUser />
+                                }
+                            </LeftBlock>
+                            <MenuHeaderBlock>
+                                <NavLink to="home"
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'rgb(133, 133, 152)' } }}>
```

Comment:
is it possible to avoid inline styles?

Reply:
Yes, of course) But this will lead to an increase in code. The Header.styled file will increase, and the number of imports will increase...

Reply:
Yep this is a problem too, but we already have an overloaded component. It could be fixed by separating code on a few components